homeassistant/components/litterrobot/sensor.py

```diff
@@ -225,3 +232,10 @@ def icon(self) -> str | None:
         if (icon := self.entity_description.icon_fn(self.state)) is not None:
             return icon
         return super().icon
+
+    @property
+    def last_reset(self) -> datetime | None:
+        """Return the time when the sensor was last reset, if any."""
+        if self.entity_description.key == "visits_today":
+            return dt_util.start_of_local_day()
+        return super().last_reset
```

Comment:
not strictly necessary, and feel free to leave it for a later PR, but it would be cool to have this as part of `RobotSensorEntityDescription` and then here just do:

```python
return self.entity_description.last_reset() or super().last_reset()
```

Reply:
Done!